src/dbt_client/index.ts

```diff
@@ -297,7 +297,11 @@ export class DBTClient implements Disposable {
 
   private async handlePythonExtension(): Promise<void> {
     const pythonEnvironment = await this.pythonEnvironment.getEnvironment();
-    this.pythonPath = pythonEnvironment.getPythonPath();
+    this.pythonPath = getPythonPathFromConfig() || pythonEnvironment.getPythonPath();
     await this.checkAllInstalled();
   }
 }
+
+function getPythonPathFromConfig(): string | undefined {
```

Comment:
Move to separate PR.

Reply:
I think we still need this. I've made arguments already in its favor and I think its better to have an escape hatch for people who need it if it doesn't fundamentally affect 80% of users except those that need/want it. 

I think I am doubly motivated to keep this other than the fact its so minor, in that it helps me test the extension at a reasonable scale; not a toy project, but on multiple real production projects which benefits our iteration cycle with a heavy dose of UAT.

Reply:
As a compromise I made the verbiage extremely clear that most users wont need to touch this.

> Path to a python executable or entrypoint. If undefined, we will use the Python extension configured environment. Most users will not need to change this setting without specific reasoning such as custom wrappers and should instead modify the Python extensions selected interpreter.